docker-compose.test.yaml

```diff
@@ -1,21 +1,23 @@
 version: '3.8'
 services:
-    service-base: # rename this to something like <yourservicename>-service
+    feedback-portal: # rename this to something like <yourservicename>-service
         build: .
         volumes:
             - ./src:/usr/app/src
         ports:
             - '3000:3000'
         command: yarn test
         ###### uncomment below as appropriate
-        # depends_on:
-        #     - mongo
+        depends_on:
+            - mongo
         #     - postgresql
         #     - rabbitmq
         #     - redis
     ###### uncomment the correponsing db's/services you need
-    # mongo:
-    #     image: mongo:4.2.8-bionic
+    mongo:
+        image: mongo:4.2.8-bionic
+        ports:
```

Comment:
Lets change the ports here in this file and the `docker-compose.test.yaml`.  If someone has a locally running mongo instance (as I do), then there is a conflict.